src/ParticleContainer.cpp

```diff
@@ -0,0 +1,77 @@
+//! A Container class for Particle Objects
```

Comment:
The docstrings are fine, but there is one major flaw:

If you build a library, you usually also distribute the headers with it - not the source files.

Hence, doc-comments should **always** be placed in the header file `*.h`, not the source file `*.cpp` for example, in case of `Particle.h|cpp`